shared/chat/conversations-list/index.desktop.js

```diff
@@ -40,19 +40,22 @@ const Row = ({onSelectConversation, selectedConversation, onNewChat, nowOverride
   const participants = participantFilter(conversation.get('participants'))
   const isSelected = selectedConversation === conversation.get('conversationIDKey')
   const isMuted = conversation.get('muted')
+  const hasUnread = !!conversation.get('unreadCount')
   // $FlowIssue
   const avatarProps = participants.slice(0, 2).map((p, idx) => ({
     backgroundColor: globalColors.darkBlue4,
     username: p.username,
-    borderColor: rowBorderColor(idx, Math.min(2, participants.count()) - 1, !!conversation.get('unreadCount'), isSelected),
+    borderColor: rowBorderColor(idx, Math.min(2, participants.count()) - 1, hasUnread, isSelected),
   })).toArray()
   const snippet = conversation.get('snippet')
-  const subColor = isSelected ? globalColors.white : globalColors.blue3_40
+  const subColor = (isSelected || hasUnread) ? globalColors.white : globalColors.blue3_40
```

Comment:
There are effectively 3 states now. Selected, not selected, not selected + badged